src/extension.ts

```diff
@@ -132,6 +150,8 @@ export function activate(context: ExtensionContext): SchemaExtensionAPI {
     client.onRequest(VSCodeContentRequest.type, (uri: string) => {
       return getJsonSchemaContent(uri, schemaCache);
     });
+
+    telemetry.send({ name: 'server_initialized', type: 'track' });
```

Comment:
yaml.server.initialized

Reply:
Actually `type` is optional, set to `track` by default in https://github.com/redhat-developer/vscode-redhat-telemetry/blob/main/src/services/reporter.ts#L35